app/controllers/user_reductions_controller.rb

```diff
@@ -0,0 +1,71 @@
+class UserReductionsController < ApplicationController
```

Comment:
Wondering if we need this, or if we can get away with just the GraphQL API for the new stuff (keeping the old routes in place for the subjects)

Reply:
I would like to keep both. I personally find GraphQL quite inferior to REST in many of the same ways that SOAP was and I've never quite been able to wrap my head around how to get it working properly. Plus there's much more tooling for consuming RESTful APIs than GraphQL APIs. Just my vote, though.